extensions/pom.xml

```diff
@@ -20,7 +20,7 @@
     <jstl.version>1.0.5</jstl.version>
     <license.header.definition.file>${basedir}/../license/styles/javadoc_style_license_header.xml</license.header.definition.file>
     <hamcrest-library.version>1.3</hamcrest-library.version>
-    <tika-parser.version>1.12</tika-parser.version>
+    <tika-parser.version>1.22</tika-parser.version>
```

Comment:
Same here. There are several declared `tika` artifacts that are not used in our code.
Please refactor these.